### PR TITLE
[MOBILE-2407] Adding a two line toast.

### PR DESCRIPTION
**Pull request by jeffscaturro-wf**
Description
---
We needed the ability to have a toast with two separate lines of text (not just using a new line character), which would allow for the two lines respective labels to be truncated differently (i.e. top could be a tail truncation where the bottom could be a middle truncation).

What Was Changed
---
* Subclassed `WToastView` with the new class and made the necessary changes to support two labels (two lines of text).
* Bumped podfile.lock to use 1.2.0.beta.1 version of cocoapods for smithy to match wdesk-ios.

Testing
---
- [x] Example project has a new toast button that demonstrates this. 
- [x] Necessary unit test coverage.
- [x] Smithy succeeds.
- [x] Smithy still shows test results & test artifacts.

---

Please Review: @Workiva/mobile

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

File: Source/WToast.swift [hunk 1]
```diff
@@ -350,3 +350,107 @@ public class WToastView: UIView {
...
+        rightIconImageView.snp_remakeConstraints { (make) in
+            make.centerY.equalTo(self)
+            make.right.equalTo(self).offset(-frame.size.width / 10)
+            make.height.equalTo(14)
+            make.width.equalTo(14)
+        }
+        rightIconImageView.image = rightIcon
+
+        firstLabel.snp_remakeConstraints { (make) in
+            make.top.equalTo(self).offset(8)
```
Comment (jamesromo-wf) [inline Source/WToast.swift]: We will want to probably make this customizable at some point since the labels can be changed and the consumer will not be able to change the constraints. (same with the second label's bottom spacing). Up to you if you want to do it now or later.

The left and right we can do as well but it's not as important since have the truncating stuff.

File: Source/WToast.swift [hunk 2]
```diff
@@ -350,3 +350,107 @@ public class WToastView: UIView {
...
+            make.height.equalTo(frame.size.height/2)
```
Comment (jamesromo-wf) [inline Source/WToast.swift]: Is this too big since it is not accounting for the top and bottom padding?

File: Source/WToast.swift (hunk 1 above)
Comment (jeffscaturro-wf) [inline Source/WToast.swift]: I was following current `WToast` implementation, since that would need to be refactored as well, I suggest doing that in a follow up ticket as we would want to revisit that class as well to support said customization. 

Logged [MOBILE-2415](https://jira.atl.workiva.net/browse/MOBILE-2415)

File: Source/WToast.swift (hunk 2 above)
Comment (jeffscaturro-wf) [inline Source/WToast.swift]: Good catch, it didn't appear wrong visually, but the frames were indeed too large. Pushing a fix.

**Comment by jamesromo-wf** [comment]
+10

- [ ] Appropriate reviews.
- [ ] Unit test created or updated.
- [ ] Acceptance criteria met.
- [ ] Edge cases tested.

@Workiva/release-management-p

**Comment by teresarevious-wf** [comment]
RM +1

verified dependencies manually
